packages/gatsby-theme-newrelic/README.md

```diff
@@ -943,6 +985,85 @@ import { NewRelicLogo } from '@newrelic/gatsby-theme-newrelic';
 <NewRelicLogo />
 ```
 
+### `PageTools`
+
+Used as a "right rail" container to display content related to the current page.
+To build modules contained inside this component, see the
+[`PageTools.Section`](#pagetoolssection) and [`PageTools.Title`](#pagetoolstitle)
+documentation.
+
+```js
+import { PageTools } from '@newrelic/gatsby-theme-newrelic';
+```
+
+**Props**
+
+| Prop        | Type   | Required | Default | Description                                          |
+| ----------- | ------ | -------- | ------- | ---------------------------------------------------- |
+| `className` | string | no       |         | Additional `className` for the component.            |
+| `children`  | node   | no       |         | Content to be displayed in the `PageTools` component |
```

Comment:
Do we want to explicitly note that Section contents takes HTML and not markdown?

Reply:
Technically `children` could be anything so it really depends on where the `PageTools` component is rendered. If this happened to be used in an MDX file (we shouldn't in practice) then markdown would render just fine. Since we don't call this out for any other component, I wouldn't specify this as one way or the other.